libs/community/langchain_community/tools/arxiv/tool.py

```diff
@@ -25,7 +26,8 @@ class ArxivQueryRun(BaseTool):
         "Computer Science, Quantitative Biology, Quantitative Finance, Statistics, "
         "Electrical Engineering, and Economics "
         "from scientific articles on arxiv.org. "
-        "Input should be a search query."
+        "Input should be a search query. If there is something related to date please use date format and not words! "
+        f"Today's date is {datetime.today().strftime('%Y-%m-%d')}."
```

Comment:
This doesn't work on any production server that runs for more than a single day

Reply:
Thanks for the quick response. You are right so for now I am reverting my changes.

On another note the input query grammar for the tool is not specified, but this I will leave for now and maybe create a separate pull request in future.